src/parthenon_manager.cpp

```diff
@@ -28,16 +28,18 @@
 
 #include "amr_criteria/refinement_package.hpp"
 #include "config.hpp"
-#include "driver/driver.hpp"
+#include FS_HEADER
```

Comment:
I don't recall---Why is this needed? Is this related to `std::filesystem`? If so, I suggest we might want to avoid using it... as it's been problematic on some machines.

Reply:
It's needed for checking the end of the filename and addresses the issue with `std::experimental::filesystem` versus `std::filesystem` for some old compilers/stdlibs.

Are you suggesting to remove it?

Reply:
We've had trouble with `std::filesystem` on some lanl systems, so I lean towards removing that library from our dependencies... but that's maybe a longer discussion and doesn't belong in this PR.

Reply:
(i.e., the discussion doesn't belong in this PR. your changeset can stay.)